services/QuillLMS/client/app/bundles/Staff/helpers/evidence/ruleHelpers.tsx

```
@@ -418,15 +418,15 @@ export async function handleSubmitRule({
 }
 
 export function getRulesUrl(activityId: string, promptId: string, ruleType: string) {
-  const url = `activities/${activityId}/rules`;
-  if(promptId && !ruleType) {
-    return `rules?prompt_id=${promptId}`
-  } else if(!promptId && ruleType) {
+  if (activityId) {
+    return `activities/${activityId}/rules`
+  } else if (!ruleType) {
+    throw new Error('A rule type must be specified.')
```

This code path is not currently hit; it's meant to inform future developers. I'm open to alternatives, like 
- returning a null string
- catching the error in the caller context.
- don't document anything here, and just let the controller throw an exception.

---

Hmm, it might be better to have the controller throw this error since our backend reporting is a little noisier. It's a staff endpoint, so it's not the biggest deal either way.

That might also allow you to turn this code branching into just passing along `prompt_id` and `rule_type` params to the url and letting the backend deal with it.

---

FYI The controller currently throws an error as well, via strong params. In light of this, I'll keep this slightly redundant front end check for now (my bias leans toward code clarity and intentionality).